packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -639,8 +639,10 @@ export const ContractType = (() => {
     const getCancellation = (contract_type: string, cancellation_duration: string, symbol: string) => {
         const arr_cancellation_range: string[] =
             getPropertyValue(available_contract_types, [contract_type, 'config', 'cancellation_range']) || [];
-
+        const cached_arr_cancellation: string[] =
+            getPropertyValue(available_contract_types, ['multiplier', 'config', 'cancellation_range']) || [];
         const regex = /(^(?:\d){1,})|((?:[a-zA-Z]){1,}$)/g;
+        const mapCancellationRangeList = (d: string) => ({ text: `${getText(d)}`, value: d });
```

Comment:
@kate-deriv please put i `mapCancellationRangeList` declaration after `getText` declaration just to be on the safe side :)
Also, I was thinking to rename `cached_arr_cancellation` into `cached_multipliers_cancellation`, so that we accentuate that it's mainly an array for `multipliers` since we are extracting it from the object with 'multiplier' data at Line 643.

Reply:
Will do